src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
 
 Moreover, it is also important to notice that the Scheduler and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same scheduler database since they use that one.
 
+==== Example configuration with PostgreSQL
+It is recommanded to use a PostgreSQL database in a ProActive prod environment.
+In the following, we show an example of PostgreSQL installation and configuration to be used along with ProActive.
+
+===== PostgreSQL version
+PostgreSQL version 12.2 or 12.9 is required to be installed. PostgreSQL could be installed on the same machine as for ProActive or on a separate server (on-prem or on the cloud).
+
+===== Initialize the database
+To initialize the DB, some SQL queries should be executed in the PostgreSQL server machine.
+To do so, please create the init script in the /tmp folder for example and put the following SQL queries:
+....
+-- Create users
+CREATE USER rm WITH PASSWORD 'rm';
+CREATE USER scheduler WITH PASSWORD 'scheduler';
+CREATE USER catalog WITH PASSWORD 'catalog';
+CREATE USER pca WITH PASSWORD 'pca';
+CREATE USER notification WITH PASSWORD 'notification';
+
+-- Creating empty databases
+CREATE DATABASE rm OWNER rm;
+CREATE DATABASE scheduler OWNER scheduler;
+CREATE DATABASE catalog OWNER catalog;
+CREATE DATABASE pca OWNER pca;
+CREATE DATABASE notification OWNER notification;
+
+-- Grant access
+GRANT CONNECT ON DATABASE scheduler TO scheduler;
+GRANT USAGE ON SCHEMA public TO scheduler;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO scheduler;
+GRANT CONNECT ON DATABASE rm TO rm;
+GRANT USAGE ON SCHEMA public TO rm;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO rm;
+GRANT CONNECT ON DATABASE catalog TO catalog;
+GRANT USAGE ON SCHEMA public TO catalog;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO catalog;
+GRANT CONNECT ON DATABASE pca TO pca;
+GRANT USAGE ON SCHEMA public TO pca;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO pca;
+GRANT CONNECT ON DATABASE notification TO notification;
+GRANT USAGE ON SCHEMA public TO notification;
+GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA public TO notification;
+
+ALTER SYSTEM SET max_connections ='500';
+....
+
+Set the host name or IP address(es) to listen to:
+....
+sudo vim /opt/postgres/data/postgresql.conf
+listen-addresses = ‘IP_ADDRESS’
+....
+
+Then, execute the script
+....
+psql -f /tmp/init.sql
+....
+
+===== Prepare the database purge operation
+In normal PostgreSQL operation, tuples that are deleted or obsoleted by an update are not physically removed from their table;
+they remain present until a purge (VACUUM) is done. Therefore it is necessary to do a purge periodically, especially on frequently-updated tables.
+In our case, the preparation for the DB purge operation is done in two steps.
```

Comment:
Prior to execute the above operations, a preliminary configuration step is necessary (this preliminary step is done once for a given ProActive installation).